src/ripple/rpc/handlers/AccountTx.cpp

```diff
@@ -157,27 +157,26 @@ Json::Value doAccountTx (RPC::Context& context)
                 *account, uLedgerMin, uLedgerMax, bForward, resumeToken, limit,
                 isUnlimited (context.role));
 
-            for (auto& it: txns)
+            for (auto& [txn, txMeta]: txns)
```

Comment:
auto const&

Reply:
Fixed in a225b62e5 `Add const to some structured bindings`